constructors/README.md

```diff
@@ -0,0 +1,3 @@
+# constructors
+
+This contains the basic folder structure for a module of the extension.
```

Comment:
Maybe you can add here the structure even if it's the default text. bq/ library/... you can grab it from other modules.

Reply:
Great, i'll update the PR accordingly.